src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -32,6 +39,11 @@ public Subscription getAppellantSubscription() {
         return responseWrapper.getNewSscsCaseData().getSubscriptions().getAppellantSubscription();
     }
 
+    @Override
+    public Subscription getRepresentativeSubscription() {
+        return responseWrapper.getNewSscsCaseData().getSubscriptions().getRepresentativeSubscription();
```

Comment:
A check should be added to fine whether representative subscription exists or not.

Reply:
as explained above the subscription object is always built even if in the json is not present.